asf_core_data/pipeline/mcs/process/process_mcs_installations.py

```diff
@@ -117,7 +120,22 @@ def identify_clusters(hps, time_interval=base_config.MCS_CLUSTER_TIME_INTERVAL):
     return hps
 
 
-#####
+def get_installer_unique_id(installations, installers):
+    """
+    Updates installations table by adding the unique installer ID.
+    Args:
+        installations: installations table
+        installers: historical installers table
```

Comment:
maybe good to mark what dtype is expected, either in doc string or type hint

Reply:
good shout, completely missed it! :D